recipes/graphlan/meta.yaml

```diff
@@ -3,7 +3,7 @@ about:
     summary: "GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation."
 package:
   name: graphlan
-  version: '1.0.0'
+  version: '0.9.7'
```

Comment:
The version number is different in scripts. But I can maintain 1.0.0.
This update was more for `matplotlib`, to avoid a warning